packages/toolkit/src/lib/vdp-sdk/organization/actions.ts

```diff
@@ -0,0 +1,65 @@
+import { MEMBERSHIP_STATE, Membership, ROLE } from "./types";
+import { createInstillAxiosClient } from "../helper";
+import { Nullable } from "../../type";
+
+export type AcceptOrganizationMembershipPayload = {
+  role: MEMBERSHIP_STATE;
+};
+
+export type MembershipResponse = {
+  membership: Membership;
+};
+
+export type UpdateOrganizationMembershipRolePayload = {
+  role: ROLE;
+};
+
+export async function updateOrganizationMembershipRoleAction({
+  payload,
+  accessToken,
+  organizationName,
+  userName,
+}: {
+  payload: UpdateOrganizationMembershipRolePayload;
+  accessToken: Nullable<string>;
+  organizationName: string;
+  userName: string;
```

Comment:
@iamnamananand996 I will consider putting them in the payload if that is all needed to operate this action and then remove them like this 

```
payload: {
  ...payload
  organizationName: undefined,
  userName: undefined
}
```